docker/build/build.sh

```diff
@@ -21,12 +21,15 @@ docker run --name orbs_build orbs:build sleep 1
 
 export SRC=/src
 
-rm -rf _bin
+rm -rf _bin && mkdir -p _bin
+rm -f ./_bin/go.mod.template
+cp ./docker/build/go.mod.template ./_bin/go.mod.template
```

Comment:
maybe replace `_bin` with `docker_build` or something that will also be added to gitignore